core/types/arbitrum_signer.go

```diff
@@ -40,6 +41,20 @@ func (s arbitrumSigner) Sender(tx *Transaction) (common.Address, error) {
 		}
 		fakeTx := NewTx(&legacyData.LegacyTx)
 		return s.Signer.Sender(fakeTx)
+	case *ArbitrumSubtypedTx:
+		switch inner.TxData.(type) {
+		case *ArbitrumTippingTx:
+			V, R, S := tx.RawSignatureValues()
```

Comment:
3 options, I'm o.k. with any:
1) add a comment that this comes from london signer as these are assentially DynamicFeeTxType
2) create a fake DynamicFeeTx and use the internal signer, similar to ArbitrumLegacyTx
3) separate the code that extract the sender inside london-signer (without checking tx type), call this func from both london signer after checking type and from here.

Reply:
2 - we can't make fake `DynamicFeeTx` and pass it to `londonSigner` because `Hash` method must be called on `arbitrumSigner` to include transaction subtype in hashed data
3 - probably would create some unnecessary merge conflicts against upstream

so I went with 1 and added a comment. 

In `SignatureValues` method I used the fake tx pattern, as there it was possible.